src/content/reference/k8s-metadata/labels/_index.md

```diff
@@ -0,0 +1,153 @@
+
+Name of the Giant Swarm team responsible for the application.
+
+### chart-operator.giantswarm.io/version
+
+Labels in the form `OPERATOR_NAME.giantswarm.io/version` are used to specify the version of the operator that should reconcile the resource. The operator name is the name of the operator that should reconcile the resource, and the version is the version of the operator that should reconcile the resource.
+
+If the operator name is `chart-operator`, the label is `chart-operator.giantswarm.io/version`. The value of the label is a SemVer version number. The special value `0.0.0` means that the resource is handled by the "unique" operator, which is an instance of the operator deployed to the `giantswarm` namespace of the cluster.
+
+[Source](https://github.com/giantswarm/k8smetadata/blob/v0.24.0/pkg/label/version.go#L13)
+
+### cluster-apps-operator.giantswarm.io/watching
+
+The label of the format `OPERATOR_NAME.giantswarm.io/watching` is used on resources to enable the operator to watch the resource. The operator name is the name of the operator that should watch the resource. If the operator name is `cluster-apps-operator`, the label is `cluster-apps-operator.giantswarm.io/watching`. The value of the label must be `true`.
+
+[Source](https://github.com/giantswarm/k8smetadata/blob/v0.24.0/pkg/label/app.go#L9)
+
+### giantswarm.io/aws-ebs-limit
+
+This label is set on EBS CSI driver pods (on AWS) by [aws-ebs-csi-volume-limiter](https://github.com/giantswarm/aws-ebs-csi-volume-limiter/blob/v0.1.0/main.go#L86-L90). It informs about the maximum number of EBS volumes that can be attached to the respective node.
+
+### giantswarm.io/cluster
+
+Used on a variety of resources to associate the resource with a specific cluster. The value is the name of the cluster.
+
+### giantswarm.io/logging
+
+With this label on the Cluster resource, logging via Loki can be disabled. The default value `true` means that logging is enabled. Setting it to `false` disables logging for the entire cluster.
+
+### giantswarm.io/machine-pool
+
+Associates the resource or the node with a node pool, using the name of the node pool as the value.
+
+### giantswarm.io/managed-by
+
+- TODO Found on various resources in MC and WC. Value differs.
+    - E. g. `rbac-operator` on RBAC resources indicates that rbac-operator created these and will reconcile/delete them.
+    - Value `flux` on an App resource indicates that the App is reconciled by Flux.
+    - Name of a bundle on an App resource indicates that the App has been installed by a bundle.
+
+### giantswarm.io/monitoring
+
+This label set on the Cluster resource with the value `false` can be used to disable the gathering of Prometheus metrics for the entire cluster, and id prevents deployment of Prometheus for this cluster.
+
+On other resources, the label serves as a discovery mechanism for metrics scraping. We're replacing this mechanism by service monitors.
+
+### giantswarm.io/monitoring_basic_sli
```

Comment:
This label is getting removed as part of https://github.com/giantswarm/giantswarm/issues/31095

Reply:
Can you remove it already?

Reply:
cc @marians :)

Reply:
Done